SimPEG/utils/drivers/gravity_driver.py

```diff
@@ -5,7 +5,7 @@
 
 try:
     from SimPEG import utils
-except:
+except ImportError:
```

Comment:
I'm not even sure how these lines made it into here, but I think it's safe to remove the try and just straight up do:
`from SimPEG import utils`